protos/sources/vega/chain_events.proto

```diff
@@ -20,8 +20,8 @@ message BuiltinAssetWithdrawal {
   string vega_asset_id = 1;
   // Vega network party ID i.e. public key.
   string party_id = 2;
-  // The amount to be withdrawn
-  // This field is an unsigned integer passed as a string and needs to be scaled using the asset's decimal places.
+  // The amount to be withdrawn. This field is an unsigned integer passed as a string
```

Comment:
Maybe @gordsport has an opinion, but for all of these can we go with a phrasing more like
"Must represent an unsigned integer scaled to the asset's decimal places." ? For me it flows a bit better.

Reply:
I made the suggested edits and marked the other conversations resolved, the edits can be found in this commit https://github.com/vegaprotocol/vega/pull/8163/commits/6839a413141d2f91a205dedf35f6346084a65e31